### PR TITLE
Bump v1.2.0

### PR DESCRIPTION
**Pull request by aashishgurung**
# Objective

Updated metadata for v1.2.0

## What's changed
- Migrate dep to go modules. (PR [#69](https://github.com/omise/omise-go/pull/69))
- Update README to fix example and update to Opn Payments. (PR [#70](https://github.com/omise/omise-go/pull/70))

### CODE REVIEW

**Review by ajzkk — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,6 +22,13 @@ Auditor.**
 Instead we recommend that you follow our guide on how to safely
 [collect credit information](https://docs.opn.ooo/collecting-card-information).
 
+## Compatibility
+
+The version `1.2.0` is designed to work with Go version 1.16 or higher. It is not compatible with Go versions 1.15 and below.
+
+If you are using an older version of Go, please consider upgrading to a compatible version to use this project effectively.
+
+
```
Comment (danfowler) [inline README.md:31]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -22,6 +22,13 @@ Auditor.**
 Instead we recommend that you follow our guide on how to safely
 [collect credit information](https://docs.opn.ooo/collecting-card-information).
 
+## Compatibility
+
+OmiseGO version `v1.2.0` and up is designed to work with Go version 1.16 or higher. It is not compatible with Go versions 1.15 and below.
```
Comment (danfowler) [inline README.md:27]: I'd rather not use the name "OmiseGO"; instead I would say "version `v1.2.0` and up of this package..."

Comment (aashishgurung) [reply README.md]: Thanks. Updated.

**Review by AnasNaouchi — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -22,6 +22,12 @@ Auditor.**
 Instead we recommend that you follow our guide on how to safely
 [collect credit information](https://docs.opn.ooo/collecting-card-information).
 
+## Compatibility
+
+The version `v1.2.0` and up of this package is designed to work with Go version 1.16 or higher. It is not compatible with Go versions 1.15 and below.
```
Comment (danfowler) [inline README.md:27]:
```suggestion
Version `v1.2.0` and up of this package is designed to work with Go version 1.16 or higher. It is not compatible with Go versions 1.15 and below.
```

Comment (aashishgurung) [reply README.md]: Thanks. Done.

**Review by danfowler — APPROVED** [review]

**Pull request merged**